frontend/packages/kube-types/.tools/config.ts

```diff
@@ -0,0 +1,63 @@
+import * as process from 'process';
+
+export const TEMP_DIR = 'tmp';
+export const SRC_DIR = 'src';
+
+export const buildTempPath = (...segments) => [TEMP_DIR, ...segments].join('/');
+export const buildSrcPath = (...segments) => [SRC_DIR, ...segments].join('/');
+
+const OPENSHIFT_PREFIX = 'OS';
+
+export type Config = {
+  name: string;
+  url: string;
+  replaceMapping?: { [key: string]: string };
+  dateAsString?: boolean;
+};
+
+export const getOpenshiftConfig = (): Config => ({
+  name: 'openshift',
+  url:
+    process.env.OPENSHIFT_API_SPEC_URL ||
+    'https://raw.githubusercontent.com/openshift/origin/master/api/swagger-spec/openshift-openapi-spec.json',
+  dateAsString: true,
```

Comment:
I removed some package names from generated object names because the names would be too long (as discussed in #1773).

There are some conflicts between the different packages so I had to leave some in form of short prefixes.

Please give me some feedback what are your thoughts on this. And which packages should be namespaced and what prefixes to use.